environment.yaml

```diff
@@ -4,5 +4,7 @@ channels:
 dependencies:
   - python=3.9
   - gwpy=2.1.0
+  - astropy<5.0.0
```

Comment:
This is a bit restrictive in an environment like this right?

Reply:
yeah thats a good point. I *think* there was a reason for this but I'll try to loosen the restrictions.

Reply:
There was an issue when gwpy 2.1 first dropped that had incompatibilities with astropy>=5.0 but they may have been resolved by now (and not just dependency collisions, the import itself would break)

Reply:
And one of these libs (I forget who, I think it's astropy) uses an outdated numpy syntax that breaks on newer versions of numpy, which forces us to restrict numpy in some of our other projects. This is all probably good incentive to start filing issues on the project GitHubs themselves and linking to them in project issues so that we can keep tabs on all this

Reply:
It appears this is resolved in `gwpy=3.0.0`

Reply:
The numpy issue or the astropy issue, or both I guess?